src/librustc/query/mod.rs

```diff
@@ -1114,9 +1114,9 @@ rustc_queries! {
         }
 
         /// Do not call this query directly: invoke `normalize_erasing_regions` instead.
-        query normalize_ty_after_erasing_regions(
-            goal: ParamEnvAnd<'tcx, Ty<'tcx>>
-        ) -> Ty<'tcx> {
+        query normalize_generic_arg_after_erasing_regions(
+            goal: ParamEnvAnd<'tcx, crate::ty::subst::GenericArg<'tcx>>
```

Comment:
importing `GenericArg` seems to fail for whatever reason

Reply:
You need to do it in `src/librustc/ty/query/mod.rs` I believe, there's some weird macro shenanigans going on.